jest.config.js

```diff
@@ -1,7 +1,8 @@
 const config = {
     preset: 'ts-jest',
     verbose: true,
-    transform: {}
+    transform: {},
+    testPathIgnorePatterns: ['dist'],
```

Comment:
running `npm test` after `npm run build` would make Jest try to run the `.test.js` in the dist folder.